app/src/main/java/src/MovePiece.java

```diff
@@ -6,5 +6,5 @@
 
 public interface MovePiece {
     void move(int posX, int posY, Piece piece);
-    public List<Position> getValidMoves(Position position);
+    public List<Position> getValidMoves(Position position, boolean colorPiece);
```

Comment:
Not sure about adding a color piece parameter, since you need the piece from the given position u can just get the color of the piece accessing the piece in the board from the given position.

Reply:
that is right, thanks